src/device-registry/validators/site.validators.js

```
@@ -0,0 +1,499 @@
+
+  // Validate numeric fields
+  const numericFields = ["latitude", "longitude", "search_radius"];
+  let isValid = true;
+
+  numericFields.forEach((field) => {
+    if (!(field in value)) {
+      isValid = false;
+      return;
+    }
+    const numValue = parseFloat(value[field]);
+    if (isNaN(numValue)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)